ISICbalance/AccountViewController.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  AccountViewController.swift
+//  ISICbalance
+//
+//  Created by Rostislav Babáček on 11/03/2019.
+//  Copyright © 2019 Rostislav Babáček. All rights reserved.
+//
+
+import UIKit
+
+class AccountViewController: UIViewController {
+
+    let accountView = AccountView()
+
+    override func loadView() {
+        super.loadView()
+        self.view = accountView
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        self.navigationController?.isNavigationBarHidden = false
+        self.navigationItem.title = L10n.Login.title
+    }
+    
+    override func viewWillDisappear(_ animated: Bool) {
+        self.navigationController?.isNavigationBarHidden = true
```

Comment:
Tahle kombinace `isNavigationBarHidden` ve viewDidLoad a willDisappear ti podle mě nemůže správně fungovat 🤔 Až se někdy pokáme tak si vzpomeň, vysvětlím jak je to správně.

Reply:
S navbarem a tabbarem jsem dost zápasil i v semestrálce. Dost jsem to googlil, ale nikdy nenařel řešení, které by fungovalo 🤷🏻‍♂️ Zapíšu si do notes.

Reply:
Lukáš mi poradil, ať se podívám do toho objc kódu a už jsem to vyřešil podle toho. :)